packages/shopware-6-client/src/categoryService.ts

```diff
@@ -22,7 +22,7 @@ export interface CategoryService {
 }
 
 const getCategories = async function(): Promise<SearchResult<Category[]>> {
-  const resp = await axios.get(config.endpoint + getCategoryEndpoint());
+  const resp = await apiService.get(config.endpoint + getCategoryEndpoint());
```

Comment:
Maybe we can use `getXEndpoint()` methods to generate full URLs? This way we will not have to use `config.endpoint` in every method. Not required to do that within' that one pull request but it is an idea for another issue. If you agree with that then we can create separate issue for that.

Reply:
even better, I added `reloadConfiguration` method, so we don't have to pass endpoint, but we still can :) I just forgot to remove it here, corrected now!